pagopa/io-react-native-cie-pid
# feat: [SIW-460] Add ios cieidsdk framework and swift module

### Description
This PR adds iOS support to CIE SDK. It was used official https://github.com/pagopa/io-cie-ios-sdk repository changed to managed custom PID data.

### How to test
Start example app and try to scan a CIE to read related user data. I should work both iOS and Android.
